client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -23,143 +23,172 @@
 
-        <!-- Speakers column -->
-        <ng-container matColumnDef="speakers">
-            <mat-header-cell *matHeaderCellDef mat-sort-header>Speakers</mat-header-cell>
-            <mat-cell *matCellDef="let motion">
-                <button mat-icon-button (click)="onSpeakerIcon(motion, $event)" [disabled]="isMultiSelect">
-                    <mat-icon
-                        [matBadge]="motion.agendaSpeakerAmount > 0 ? motion.agendaSpeakerAmount : null"
-                        matBadgeColor="accent"
-                    >
-                        mic
-                    </mat-icon>
-                </button>
-            </mat-cell>
-        </ng-container>
+                <!-- Speakers column -->
+                <ng-container matColumnDef="speakers">
+                    <mat-header-cell *matHeaderCellDef mat-sort-header>Speakers</mat-header-cell>
+                    <mat-cell *matCellDef="let motion">
+                        <button mat-icon-button (click)="onSpeakerIcon(motion, $event)" [disabled]="isMultiSelect">
+                            <mat-icon
+                                [matBadge]="motion.agendaSpeakerAmount > 0 ? motion.agendaSpeakerAmount : null"
+                                matBadgeColor="accent"
+                            >
+                                mic
+                            </mat-icon>
+                        </button>
+                    </mat-cell>
+                </ng-container>
 
-        <mat-header-row *matHeaderRowDef="getColumnDefinition()"></mat-header-row>
-        <mat-row
-            [ngClass]="selectedRows.indexOf(row) >= 0 ? 'selected' : ''"
-            (click)="selectItem(row, $event)"
-            *matRowDef="let row; columns: getColumnDefinition()"
-            class="lg"
-        >
-        </mat-row>
-    </mat-table>
+                <mat-header-row *matHeaderRowDef="getColumnDefinition()"></mat-header-row>
+                <mat-row
+                    [ngClass]="selectedRows.indexOf(row) >= 0 ? 'selected' : ''"
+                    (click)="selectItem(row, $event)"
+                    *matRowDef="let row; columns: getColumnDefinition()"
+                    class="lg"
+                >
+                </mat-row>
+            </mat-table>
 
-    <mat-paginator class="on-transition-fade" [pageSizeOptions]="pageSize"></mat-paginator>
+            <mat-paginator class="on-transition-fade" [pageSizeOptions]="pageSize"></mat-paginator>
+        </span>
+        <span *ngSwitchCase="'tiles'">
+            <os-grid-layout title="{{ 'Favorites' | translate }}">
+                    <os-block-tile (clicked)="navigateTo(null, true)" [orientation]="'horizontal'" [block]="'Fav'" [blockType]="'text'" title="{{ 'Favorite motions' | translate}}">
+                        <ng-template #contentNode>
+                            <div translate>
+                                Containing motions: <mat-chip>{{ motionsAsFavorite().length }}</mat-chip>
```

Comment:
Narrow down the closure of translation: `<span translate> Containing motions</span>:`. Same below.